src/components/DateRangePicker/DefaultRanges.js

```diff
@@ -28,27 +34,25 @@ class DefaultRanges extends React.Component {
               className={`${css(styles.rangeOption)} mx-default-ranges-range`}
               key={range.displayValue + range.getStartDate()}
               onClick={() => {
-                handleDefaultRangeSelection(range);
-                this.setState({ selectedOption: index });
+                handleDefaultRangeSelection(range)
+                this.setState({ selectedOption: index })
               }}
             >
               <div>
                 <Icon
                   size={20}
-                  style={{ ...styles.rangeOptionIcon,
```

Comment:
As far as I can tell, removing the `rangeOptionIcon` style rules is the only actualy change in this PR. Is that right?

Reply:
Everything else seems to be the package updates for the CI warnings or prettier changes to formatting.

Reply:
No, there are changes in other files too. I just forgot to not save with Prettier again.

Reply:
There, go rid of Prettier formatting.

Reply:
Was it intentional to get rid of the `rangeOptionIcon` style rules?

Reply:
Answered in Slack - variable was undeclared.

Reply:
We discussed in private, and it's all just updating code. Should work the same in production.